### PR TITLE
PCHR-2032: Move outlined buttons styles to a partial

### PR DESCRIPTION
**Pull request by igorpavlov-zz**
## Overview 

This PR moves styles for outlined buttons to a separate file in order to:
1. Make other themes able to include only outlined buttons styles without importing all button styles.
2. Improve the structure of Shoreditch theme.

### CODE REVIEW

File: scss/bootstrap/overrides/style/_buttons.scss [hunk 1]
```diff
@@ -103,45 +103,4 @@
...
-}
-
-.btn-warning-outline {
-  @include button-outline-variant($btn-warning-bg);
-}
-
-.btn-danger-outline {
-  @include button-outline-variant($btn-danger-bg);
-}
+@import "../../variants/outlined-buttons";
```
Comment (AkA84) [inline scss/bootstrap/overrides/style/_buttons.scss]: you should import `variants/**/*` in the main scss file

**Review by AkA84 — CHANGES_REQUESTED** [review]

File: scss/bootstrap/variants/_outlined-buttons.scss [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  .open &,
+  &:hover,
+  &:active {
+    background-color: $crm-white !important;
+    color: $crm-dark-blue !important;
+  }
+}
+
+.btn-primary-outline {
+  @include button-outline-variant($btn-primary-bg);
```
Comment (AkA84) [inline scss/bootstrap/variants/_outlined-buttons.scss:19]: i think you need to include the mixins, otherwise you can't import this single partial

**Review by AkA84 — CHANGES_REQUESTED** [review]

File: scss/bootstrap/variants/_outlined-buttons.scss (hunk 2 above)
Comment (igorpavlov) [inline scss/bootstrap/variants/_outlined-buttons.scss]: But it works and and doesn't throw any error. Mixins are imported just before with `@import "mixins/*";`

Comment (igorpavlov-zz) [reply scss/bootstrap/variants/_outlined-buttons.scss:19]: I added mixins in the CiviHR Theme, but you are right, it is better to keep dependencies within the partial. Fixed.

**Review by AkA84 — APPROVED** [review]

**Pull request merged**